backend/src/flotilla/api/events_api.py

```diff
@@ -33,9 +34,20 @@
             "model": List[Event],
             "description": "Request successful.",
         },
+        HTTPStatus.UNAUTHORIZED.value: {
+            "model": Error,
```

Comment:
The description of return codes can be misleading since the Error model is not returned when we throw the HTTPException. Might be wrong

Reply:
What is returned is an `HTTPException` which leads to a dictionary showing this

```
{
    "detail": "some text here"
}
```

Which is what the `Error` object indicates. Not certain what else to put there as you can't indicate an `HTTPException`...